sdk/storage/azure-storage-file-datalake/src/main/java/com/azure/storage/file/datalake/implementation/util/BuilderHelper.java

```diff
@@ -39,7 +39,7 @@
  */
 public final class BuilderHelper {
     private static final String DEFAULT_USER_AGENT_NAME = "azure-storage-file-datalake";
-    private static final String DEFAULT_USER_AGENT_VERSION = "12.0.0-preview.5";
+    private static final String DEFAULT_USER_AGENT_VERSION = "12.0.0-preview.7";
```

Comment:
Would you mind adding the version update text to something like this? I missed it when I put datalake out

```
    private static final String DEFAULT_USER_AGENT_NAME = "azure-storage-blob";
    // {x-version-update-start;com.azure:azure-storage-blob;current}
    private static final String DEFAULT_USER_AGENT_VERSION = "12.1.0-beta.1";
    private static final String DEFAULT_USER_AGENT_VERSION = "12.1.0";
    // {x-version-update-end}
```

Reply:
Done